client/src/components/polls/PollModal.js

```diff
@@ -145,6 +148,14 @@ export default function PollModal(props) {
 }
 
 const useStyles = makeStyles((theme) => ({
+  dropzone: {
+    width: 350,
+    height: 300,
+    backgroundColor: '#e8e8e8',
```

Comment:
This is not an issue, just a cool thing you can do with material-ui
Instead of using a hex for gray color, you could have used:
`theme.palette.grey[100]`
You can change the 100 to be a different grey.
Check out the default theme:
https://material-ui.com/customization/default-theme/#default-theme

This way the app stays consistent with its colors.